mathics/builtin/lists.py

```diff
@@ -1683,6 +1683,94 @@ def callback(level):
 
         return Expression('List', *results)
 
+def del_part(expr,indices,evaluation):
+    def del_one(cur,pos):
```

Comment:
Is it really necessary to put a function inside a function?

Reply:
I wouldn't say it's "necessary", no.  But these are just helper functions that are not intended to be called outside the function del_part, so they are local to that scope.

Reply:
> Is that better?

I believe so.